css/styles.css

```
@@ -0,0 +1,93 @@
+/*
+- Font sizes (px):
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+
+- Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+*/
+
+:root {
+  font: 400 10px / 1.2 "Yeseva One";
+}
+
+.container {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  height: 12rem;
+  gap: 0.4rem;
+  padding-left: 18rem;
+  padding-right: 20.4rem;
+  align-items: center;
+}
+
+.logo {
+  font-size: 3.6rem;
+  text-transform: uppercase;
+  margin-right: auto;
+}
+
+.nav {
+  font-family: "Gentium Book Plus";
+  font-weight: bold;
+  align-self: stretch;
+}
+
+.nav-list {
+  display: flex;
+  gap: 2.2rem;
+  height: 100%;
+  align-items: center;
+}
+
+.nav-item {
+  position: relative;
+  display: flex;
+  align-self: stretch;
+}
+
+.nav-link {
+  font-size: 2.1rem;
+  padding-left: 1.4rem;
+  padding-right: 1.4rem;
+  align-self: center;
+}
```

In practice self alignment is rarely used, parent containers' styles do the job
```suggestion
.nav-list {
  display: flex;
  gap: 2.2rem;
  height: 100%;
  align-items: stretch;
}

.nav-item {
  position: relative;
  display: flex;
  align-items: center;
}

.nav-link {
  font-size: 2.1rem;
  padding-left: 1.4rem;
  padding-right: 1.4rem;
}
```